GuessGame.py

```diff
@@ -49,45 +48,46 @@ def main():
         print("\nSuper! Let's go!")
         print(f"The number selected is between {n1} to {n2}. Time to guess!")
         break
+
     while count <= guess_of_count:
-        while True:
-            if count >= 2:
-                hint = input("Do you want a hint? Press 'H' or 'Enter' to pass: ").lower()
-                if hint == "h":
-                    if guesses < number:
-                        print("The number was too low!")
-                        break
-                    elif guesses > number:
-                        print("The number was too high")
-                        break
-                elif hint == "":
+        if count >= 2:
+            print("Do you want a hint? Press 'H' or 'Enter' to pass: ")
+            hint = False
+            while True:
+                if keyboard.is_pressed('enter'):

```

Comment:
Why not use the keyboard here as well, I reckoned.

Reply:
I was thinking to use here as well, I just forgot to do it. Thanks to have notice!